drivers/hmis/spec/models/hmis/hud/enrollment_spec.rb

```diff
@@ -18,6 +18,49 @@
 
   include_context 'hmis base setup'
 
+  it 'detects date conflicts' do
+    [
+      # [ enter, exit, cmp enter, cmp exit, conflict expected ]
+      ['2000-01-01', '2000-01-01', '2000-01-01', nil, true, 'enter on exited entry date'],
+      ['2000-01-01', '2000-01-03', '2000-01-02', nil, true, 'enter between exited entry and exit'],
+      ['2000-01-02', '2000-01-03', '2000-01-01', nil, true, 'enter before existing entry'],
```

Comment:
I can see this being a warning, but it shouldn't be an error.  What if you were trying to do back-data input of an enrollment that occurred before another one.  Since we can't add exit and entry dates in one form, this would prevent that.